src/target/lmi.c

```diff
@@ -43,14 +46,15 @@
 #define DID0_CLASS_STELLARIS_FURY 0x00010000U
 #define DID0_CLASS_STELLARIS_DUST 0x00030000U
 #define DID0_CLASS_TIVA_C123      0x00050000U
-#define DID0_CLASS_TIVA_C129      0x000a0000U
+#define DID0_CLASS_TIVA_SNOWFLAKE 0x000a0000U
```

Comment:
According to their documentation, Snowflake is the proper family name

Reply:
Sure, not a problem - perhaps make a note of this name mapping in the source as part of the DID0 block comment?